src/worldcereal/openeo/preprocessing.py

```diff
@@ -244,7 +244,40 @@ def raw_datacube_DEM(
         fetch_type=fetch_type,
         collection_name="COPERNICUS_30",
     )
-    return extractor.get_cube(connection, spatial_extent, None)
+    """Method to get the DEM datacube from the backend.
+    If running on CDSE backend, the slope is also loaded from the global
+    slope collection and merged with the DEM cube.
+
+    Returns
+    -------
+    DataCube
+        openEO datacube with the DEM data (and slope if available).
+    """
+
+    cube = extractor.get_cube(connection, spatial_extent, None)
+    cube = cube.rename_labels(dimension="bands", target=["elevation"])
+
+    if backend_context.backend.name == "CDSE" and fetch_type == FetchType.TILE:
```

Comment:
better to do the comparison directly with the backend enum

`if backend_context.backend == Backend.CDSE`